Introduction.md

```diff
@@ -0,0 +1,85 @@
+# Introduction
+
+These standards documents describe a framework for onchain/off-chain stateful bitcoin contracts, relying on real-world observations from third-party, without leaking knowledge of usage.
+Bitcoin contracts are cooperatively built and signed
+by every party involved.
+
+## The Bet: A Short Introduction to Discreet Log Contracts
+
+DLC [were originally proposed by Taddheus Dryja](https://adiabat.github.io/dlc.pdf).
+The goal of DLC is to enable setting up contracts between two parties directly on the Bitcoin blockchain using an oracle to determine the contract outcome.
+
+### Oracle
+
+An oracle is an entity signing messages in reaction to real-world events.
+Before event occurrence the oracle advertises a public nonce that will be used to later produce a signature.
+The public key used for signing must also be known in advance, but need to be unique for each event.
+Once the event has occurred they release a signed message over the event outcome using the previously advertised nonce.
+
+An important feature of DLC is that oracle signatures can be used without an explicit request from the contract participants.
+This characteristic is the reason for the "discreet log" part of the name, as the protocol, depending on its implementation and instantiation, enables parties to hide traces of their contracts to the oracle.
+
+### Trustless execution
```

Comment:
nit: Maybe rename to something like `No Counter-party Trust`. Later in the section you say that oracles are indeed not entirely trustless

Reply:
I changed to 'trust-minimized'. It's a term we use quite a lot in CG, I feel 'No counter-Party trues' is not a really good section title.